week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -7,18 +7,26 @@
   NOTE: only the names should be returned, not the means of transport.
 */
 
-function journeyPlanner() {
-
+function journeyPlanner(arr, transport) {
```

Comment:
This logic is doing the right thing (yay!) but the tests are failing - can you work out why that is?

Also, for practice, can you think of a way to write this using filter and map, rather than two for loops? (This code totally works, but as we were learning about filter and map, it would be good to practice using them too :))